### PR TITLE
Fix price decimals

### PR DESCRIPTION
**Pull request by jonator**
### ClickUp Task

[ClickUp Task URL](PASTE_CLICKUP_TASK_URL_HERE)

### CODE REVIEW

File: packages/stores/src/queries-external/token-pair-historical-chart/index.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export class ObservableQueryTokenPairHistoricalChart extends ObservableQueryExte
     super(
       kvStore,
       baseURL,
-      `/pairs/v1/historical/${poolId}/chart?asset_in=${baseDenom}&asset_out=${quoteDenom}&range=${priceRange}&asset_type=symbol`
+      `/pairs/v1/historical/${poolId}/chart?asset_in=${quoteDenom}&asset_out=${baseDenom}&range=${priceRange}&asset_type=symbol`
```
Comment (jonator) [inline packages/stores/src/queries-external/token-pair-historical-chart/index.ts:28]: We were passing in the wrong denoms as params to the query the whole time

File: packages/stores/src/queries/concentrated-liquidity/position-by-id.ts
Comment (jonator) [inline packages/stores/src/queries/concentrated-liquidity/position-by-id.ts:1]: Adjust the price per position

File: packages/stores/src/queries/pools/pool.ts
Comment (jonator) [inline packages/stores/src/queries/pools/pool.ts:1]: Adjust the current price (not sqrt) per pool

**Comment by sunnya97** [comment]
Is fixed in pool details page, but not pools overview

CleanShot 2023-07-14 at 04 15 28@2x

**Comment by jonator** [comment]
@sunnya97 strange, that is only happening to me in previews

**Comment by sunnya97** [comment]
> @sunnya97 strange, that is only happening to me in previews

wdym?  Its not happening on local machine?

**Comment by jonator** [comment]
> > @sunnya97 strange, that is only happening to me in previews
> 
> wdym? Its not happening on local machine?

No I can't repro locally

**Comment by sunnya97** [comment]
What?  Why would that be??  Does it have to do with some query whitelists?  Or maybe something launch darkly related?

**Comment by jonator** [comment]
Preview:
Screenshot 2023-07-14 at 11 58 12 AM

Local dev:
Screenshot 2023-07-14 at 11 58 24 AM

**Comment by sunnya97** [comment]
wtf

**Review by sunnya97 — APPROVED** [review]

**Pull request merged**